### PR TITLE
Update registrymirror.md

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by cxbrowne1207 — APPROVED** [review]

File: docs/content/en/docs/getting-started/optional/registrymirror.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ spec:
   ```yaml
   ociNamespaces:
     - registry: "public.ecr.aws"
-      namespace: "eks-anywhere"
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/optional/registrymirror.md:59]: What is the implication of this? Will each registry mirror just use the default namespace? Or do you mean for them to choose their own name?

**Review by chrisnegus — COMMENTED** [review]
I just have one question.

File: docs/content/en/docs/getting-started/optional/registrymirror.md (hunk 1 above)
Comment (vivek-koppuru) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:59]: Yea I thought this is still configurable, and maybe we just explain how this maps to our registries?

**Pull request merged**